.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,165 @@
+name: "Bug Report (Package)"
+description: "Create a generic bug report against a package."
+title: "PACKAGENAME: BUG TITLE"
+labels: ["0.kind: bug"]
+body:
+  - type: "markdown"
+    attributes:
+      value: |
+        <p align="center">
+          <a href="https://nixos.org">
+            <picture>
+              <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/NixOS/nixos-homepage/main/public/logo/nixos-hires.png">
+              <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/NixOS/nixos-artwork/master/logo/nixos-white.png">
+              <img src="https://raw.githubusercontent.com/NixOS/nixos-homepage/main/public/logo/nixos-hires.png" width="400px" alt="NixOS logo">
+            </picture>
+          </a>
+        </p>
+
+        Welcome to Nixpkgs. Please replace the **`PACKAGENAME: BUG TITLE`** template above with the correct package name (As seen in the [NixOS Package Search](https://search.nixos.org/packages)) and a short title summarising what the bug entails.
+
+        > [!TIP]
+        > For instance, if you were filing a bug against the [`hello`](https://search.nixos.org/packages?channel=unstable&from=0&size=1&buckets=%7B%22package_attr_set%22%3A%5B%22No%20package%20set%22%5D%2C%22package_license_set%22%3A%5B%22GNU%20General%20Public%20License%20v3.0%20or%20later%22%5D%2C%22package_maintainers_set%22%3A%5B%5D%2C%22package_platforms%22%3A%5B%5D%7D&sort=relevance&type=packages&query=hello) package about it failing to launch on ARM Linux, your title would be as follows:
+        > `hello: fails to launch on aarch64-linux`
+
+        ---
+  - type: "dropdown"
+    id: "version"
+    attributes:
+      label: "Nixpkgs Version"
+      description: |
+        What version of NixOS/Nixpkgs are you using?
+
+        > [!IMPORTANT]
+        > If you are using an older version, please update to the latest stable version and check if the issue persists before continuing this bug report.
+      options:
+        - "Please select a version."
+        - "- Unstable (25.05)"
+        - "- Stable (24.11)"
+        - "- Oldstable (24.05)"
```

Comment:
Oldstable is a funny name, but I think "Previous stable" might work better

Reply:
I'm a bit Debian brained from that, sorry. Maybe we should use "Deprecated" since that's what the tag is on https://search.nixos.org?

Reply:
Hmm. That doesn't sound right either. I think "previous" sounds more neutral and comes off as less judgemental ("deprecated" sounds like we aren't offering any more support, which isn't necessarily true)

Reply:
**Previous Stable** it is!